package.xml

```diff
@@ -4,8 +4,7 @@
   <description>A ROS Driver for V4L USB Cameras</description>
 
   <maintainer email="andrey.vukolov@elettra.eu">Andrey Vukolov</maintainer>
-  <maintainer email="rctoris@wpi.edu">Russell Toris</maintainer>
-  <maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>
```

Comment:
i think it is better to leave ros-orphaned-packages.
ros orphaned packages group will take care of this package again if there is no maintainer in the future.